### PR TITLE
Update email palyoads

### PR DESCRIPTION
**Pull request by L3str4nge**
Related to [Saleor #9388](https://github.com/saleor/saleor/pull/9388)

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4283 Convert IDs from DB to Graphql format in Email Payload](https://app.clickup.com/t/2549495/SALEOR-4283)

File: docs/developer/available-plugins/user-emails.mdx [hunk 1]
```diff
@@ -1275,11 +1277,11 @@ Event name:  _order_fulfillment_update_
   },
   "physical_lines": [
     {
-      "id": 52,
+      "id": VXNlcjo1Mg==,
```
Comment (krzysztofwolski) [inline docs/developer/available-plugins/user-emails.mdx:1280]:
```suggestion
      "id": "VXNlcjo1Mg==",
```

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]

File: docs/developer/available-plugins/user-emails.mdx [hunk 2]
```diff
@@ -309,7 +311,7 @@ Event name: _order_confirmation_
 ```json
 {
   "order": {
-    "id": 51,
+    "id": "VXNlcjo1MQ==",
```
Comment (tomaszszymanski129) [inline docs/developer/available-plugins/user-emails.mdx:314]: that has `User` type encoded while it's an `Order` (relates to entire pr). If as a dev I read these docs and wonder what actually hides behind this magic ID, I'd immediately spot this looks like b64, decode it and get confused ;D

**Comment by maarcingebala** [comment]
@L3str4nge Since this is a breaking change we should mention it in the migration guide. Similarly as we have [3.0->3.1](https://docs.saleor.io/docs/3.x/developer/upgrade-guide/3-0-to-3-1), we should add 3.1 -> 3.2 page and mention this change with some example how the old payload looked like and how it's changed now.

**Review by tomaszszymanski129 — APPROVED** [review]

File: docs/developer/upgrade-guide/3-1-to-3-2.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+Version 3.2 introduces some breaking changes necessary to ensure the highest quality of our API. There were no changes in public APIs, so storefronts do not need any changes to be compatible.
+
+## Saleor Apps inside Dashboard
+
+### Notification payloads
+
+This change converts ID's from Database to GraphQL format in all notification payloads (email plugins and webhook.NOTIFY).
+
+Before:
+```
```
Comment (krzysztofwolski) [inline docs/developer/upgrade-guide/3-1-to-3-2.mdx:16]:
```suggestion
```json
```

File: docs/developer/upgrade-guide/3-1-to-3-2.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+{
+  "user": {
+    "id": 68,
+   ...
+}
+```
+
+After:
+```
```
Comment (krzysztofwolski) [inline docs/developer/upgrade-guide/3-1-to-3-2.mdx:25]:
```suggestion
```json
```

File: docs/developer/upgrade-guide/3-1-to-3-2.mdx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+## Preparation
+
+Version 3.2 introduces some breaking changes necessary to ensure the highest quality of our API. There were no changes in public APIs, so storefronts do not need any changes to be compatible.
+
+## Saleor Apps inside Dashboard
+
+### Notification payloads
+
+This change converts ID's from Database to GraphQL format in all notification payloads (email plugins and webhook.NOTIFY).
```
Comment (krzysztofwolski) [inline docs/developer/upgrade-guide/3-1-to-3-2.mdx:13]:
```suggestion
This change converts ID's from integers to GraphQL format in all notification payloads (email plugins and `NOTIFY` webhook).
```

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**